src/isPasswordActual.test.js

```diff
@@ -14,15 +14,31 @@ describe(`Function 'isPasswordActual':`, () => {
   });
 
   it(`should return a string`, () => {
+    const str = isPasswordActual(today.year, today.month, today.date);
 
+    expect(typeof str).toBe('string');
   });
 
-  it(`should ask to change the password if was changed a year ago`, () => {
-    const lastYear = isPasswordActual(today.year - 1, today.month, today.date);
+  // eslint-disable-next-line max-len
+  it(`should ask to change the password if was changed more than 60 days ago`, () => {
+    const lastYear = isPasswordActual(today.year, today.month - 2, today.date);
 
     expect(lastYear)
       .toBe('Immediately change the password!');
   });
 
-  // write more tests here
+  // eslint-disable-next-line max-len
+  it('should recommend to change the password if was changed a month ago', () => {
```

Comment:
```suggestion
  it(
  'should recommend to change the password if was changed a month ago',
   () => {
```
instead disable eslint better to place arguments of it function on new lines